pysmt/logics.py

```diff
@@ -594,7 +615,7 @@ def __hash__(self):
 PYSMT_LOGICS = frozenset([QF_BOOL, QF_IDL, QF_LIA, QF_LRA, QF_RDL, QF_UF, QF_UFIDL,
                           QF_UFLIA, QF_UFLRA, QF_UFLIRA,
                           BOOL, LRA, LIA, UFLIRA, UFLRA,
-                          QF_BV, QF_UFBV])
+                          QF_BV, QF_UFBV, QF_SLIA])
```

Comment:
This implies that all solvers that currently are marked as supporting PYSMT_LOGICS, need to be extended to support strings (MathSAT and Z3 for sure).
There are two options here:

* Do not put QF_SLIA in PYSMT_LOGICS, and only mark CVC4 as supporting it
* Explicitely remove QF_SLIA in all other solvers ```LOGICS``` field.

I'd prefer the first one, since so far only CVC4 supports Strings.